lib/WebpackError.js

```diff
@@ -4,8 +4,83 @@
 */
 "use strict";
 
+const createHash = require("./util/createHash");
+
+const hash = Symbol("hash");
+const meta = Symbol("meta");
+
 module.exports = class WebpackError extends Error {
+	constructor(
+		message,
+		options = Object.create({
```

Comment:
Why did you use `Object.create` instead of a plain object?

Reply:
For the same reasons you stated on your prior request?

Reply:
I talked about `Object.create(null)` to create an object with an empty prototype. In this case, it creates an object with `{…}` as prototype. Here, a plain object is better ^^

Reply:
Fixed!